app.js

```diff
@@ -0,0 +1,22 @@
+const express = require("express");
+const app = express();
+const { getTopics } = require("./servers.controllers");
+
+app.use(express.json());
```

Comment:
Please remove this middleware function.  it is only needed for requests where there is a body (this comes later - add it when you need it!)